gn3/llms/process.py

```diff
@@ -1,40 +1,74 @@
 """this module contains code for processing response from fahamu client.py"""
+# pylint: disable=C0301
 import os
+import re
 import string
 import json
-
-from urllib.parse import urljoin
-from urllib.parse import quote
 import logging
-import requests
+from urllib.parse import quote
 
 from gn3.llms.client import GeneNetworkQAClient
-from gn3.llms.response import DocIDs
 
 
 BASE_URL = 'https://genenetwork.fahamuai.com/api/tasks'
-
-
-# pylint: disable=C0301
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+
+
+class DocIDs():
+    """ Class Method to Parse document id and names from files"""
+    def __init__(self):
+        """
+        init method for Docids
+        * doc_ids.json: opens doc)ids for gn references
+        * sugar_doc_ids:  open doci_ids for diabetes references
+        """
+        self.doc_ids = load_file("doc_ids.json", BASEDIR)
+        self.sugar_doc_ids = load_file("all_files.json", BASEDIR)
```

Comment:
I don't think we use `sugar_doc_ids` outside this class, but I may be wrong. If not, we can change this to:
```
sugar_doc_ids = load_file("all_files.json', BASEDIR)
self.format_doc_ids(sugar_doc_ids)
```

Reply:
This commit fixes: https://github.com/genenetwork/genenetwork3/pull/188/commits/742beb6ee663bc9ae5409461d2be4b2144b8893e